src/server.lua

```diff
@@ -6,56 +6,71 @@ local redirecthandler = require "xavante.redirecthandler"
 
 local config = require("config")
 
--- Define here where Xavante HTTP documents scripts are located
-local webDir = "./www"
-
-local rules = {
-	
-	-- redirect
-    {
-      match = "^[^%./]*/$",
-      with = redirecthandler,
-      params = {"index.lua"}
-    }, 
-    {
-      match = "^[^%./]*/jsonrpc/?$",
-      with = redirecthandler,
-      params = {"jsonrpc.lua"}
-    }, 
-	
-	-- cgi
-    {
-      match = {"%.lp$", "%.lp/.*$", "%.lua$", "%.lua/.*$" },
-      with = cgiluahandler.makeHandler (webDir)
-    },
-	
-	-- static content
-    {
-      match = ".",
-      with = filehandler,
-      params = {baseDir = webDir}
-    },
-} 
-
-xavante.HTTP{
-    server = {host = "::", port = config.server.rpcport},
-    
-    defaultHost = {
-    	rules = rules
-    },
-}
-
-xavante.HTTP{
-    server = {host = "0.0.0.0", port = config.server.rpcport},
-    
-    defaultHost = {
-    	rules = rules
-    },
-}
-
-print "\nStarting mnigs server...\n";
+
+local mnigs_logo = '[meshnet-mgr]'
+
+-- Xavante HTTP documents scripts are located
+local webDir = config.server.webdir

```

Comment:
I think this should not be a configuration parameter.  There is no good reason why someone would change this, and if they do, they will probably break things.  What are the reason for allowing them to change it?